### PR TITLE
SDI-33 Added missing whenCreated to paged results

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt [hunk 1]
```diff
@@ -582,149 +582,150 @@ class MappingResourceIntTest : IntegrationTestBase() {
...
-
-      @Test
-      fun `access forbidden when no authority`() {
-        webTestClient.get().uri("/mapping/migration-id/2022-01-01T00:00:00")
-          .exchange()
-          .expectStatus().isUnauthorized
-      }
+  @DisplayName("GET /mapping/migration-id/{migrationId}")
+  @Nested
+  inner class GetVisitMappingByMigrationIdTest {
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt:589]: This was accidentally buried inside the DELETE test so brought it outside

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt [hunk 2]
```diff
@@ -582,149 +582,150 @@ class MappingResourceIntTest : IntegrationTestBase() {
...
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("totalElements").isEqualTo(4)
+        .jsonPath("$.content..nomisId").value(
+          Matchers.contains(
+            1, 2, 3, 4
           )
-        }
+        )
+        .jsonPath("$.content[0].whenCreated").isNotEmpty
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt:651]: The actual change - assert we get a `whenCreated` element

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt (hunk 1 above)
Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/MappingResourceIntTest.kt:589]: So nothing has actually changed in the test except the one line commented below

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**